### PR TITLE
Fix build with travis CI

### PR DESCRIPTION
**Pull request by philippgualdi**
Update setup and travis CI Build.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,32 +33,27 @@
     'pytest>=4.6.4,<5.0.0',
```
Comment (mb-wali) [inline setup.py:33]:
```suggestion
    "pytest-invenio>=1.4.0",
```
Also - remove these packages listed bellow from ```tests_require =[]```
```
'check-manifest>=0.25',
'coverage>=4.5.3',
'isort>=4.2.2',
'mock>=1.3.0',
'pydocstyle>=3.0.0',
'pytest-cov>=2.7.1',
'pytest-pep8>=1.0.6',
```
The reason for removing these packages is that we are installing the ```"pytest-invenio>=1.4.0",``` package which is installing these packages to keep the tests packages in one place. See [HERE](https://github.com/inveniosoftware/pytest-invenio/blob/798eaa4140dcdfe6e2cb51338915701a993545ee/setup.py#L48)

File: setup.py [hunk 2]
```diff
@@ -124,10 +121,9 @@
...
         'Topic :: Software Development :: Libraries :: Python Modules',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
         'Programming Language :: Python :: Implementation :: CPython',
         'Development Status :: 5 - Production/Stable',
```
Comment (mb-wali) [inline setup.py:128]:
```suggestion
       "Development Status :: 3 - Alpha",
```

Reason: the status for development is Alpha.

**Review by mb-wali — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (philippgualdi) [reply setup.py:33]: The dependency pytest-pep8 is not satisfied in pytest-invenio. Should it be removed from run-tests.sh?

Comment (mb-wali) [reply setup.py:33]: yes you can remove this package, and modify your ```pytest.ini``` file, Check [here](https://github.com/tu-graz-library/invenio-theme-tugraz/blob/master/pytest.ini) for a reference.

**Review by mb-wali — APPROVED** [review]

**Review by mb-wali — APPROVED** [review]

**Comment by mb-wali** [comment]
@philippgualdi quick check befor merging. agree if the statement below is correct.
This PR closes #1 and #5.

**Comment by philippgualdi** [comment]
Yes issue #1 and #5 can be closed.

**Pull request merged**